axum-core/src/macros.rs

```diff
@@ -26,12 +26,6 @@ macro_rules! define_rejection {
         }
 
         impl std::error::Error for $name {}
-
-        impl Default for $name {
-            fn default() -> Self {
-                Self
-            }
-        }
```

Comment:
This change seems unrelated.

Reply:
Since `BodyAlreadyExtracted` is the only rejection left in core that didn't wrap another error I thought it was nicer just add `#[derive(Default)]` directly at the definition site. Technically we could remove this whole half of the macro since its only used once.

Reply:
Okay, sure. 👍🏼 for getting rid of macro code that's only used once.

Reply:
Done!